### DISCUSSION TITLE
Consultar informações da NFe através de chave de acesso de NFe e de NFCe

### DISCUSSION THREAD

**Discussion by IlfGauhnith** [TadaSoftware/PyNFe#329] [Q&A] [answered]
Olá a todos!

Estou desenvolvendo um programa para me auxiliar na prestação de contas de determinadas compras. Tenho um CNPJ e possuo um certificado digital do mesmo.

Meu interesse é em consultar:
- Valor total da nota
- Data de emissão
- Número da nota

a partir da chave de acesso da nota.

Até agora meu programa está funcional ao consultar através da distribuição DFe para notas de modelo 55. Meu problema surge quando vou consultar uma NFCe a partir da chave de acesso. A consulta por distribuição DFe retorna: 
- cStat: **618**
- xMotivo: **Rejeicao: Chave de Acesso invalida (modelo diferente de 55)**

Como posso resolver isso? Estou utilizando o método errado? Existe um método para consultar NFe através da chave de acesso da NFCe?

**Comment by juniortada** [comment, marked as answer]
@IlfGauhnith não existe consulta DFe para NFC-e (modelo 65).
Diferente de NF-e, NFC-e depende exclusivamente da SEFAZ de cada estado.
Não existe um "ambiente nacional" mesmo existindo uma documentação orientando a implementação.
